Fixes

amtyurin opened the pull request.

Trilarion commented:
I had done a commit on master here before you started working on fixes. I would like to keep a linear history if possible. Therefore I rebased your fixes branch against my master branch and also added another commit. This means that the histories of my master and your master are now different, while the actual content is almost the same.

I advice you to:

- check that your fixes and my master is almost identical (with a directory comparison tool for example)
- reset your master to my master for example with something like
git remote add upstream https://github.com/Trilarion/imperialism-remake.git
git reset --hard master upstream/master
- continue with a new feature branch (fixes is not needed anymore or reset fixes than to your master)

Where do you prefer to discuss further programming? Here in the issues or on the forum at https://remake.twelvepm.de/forum/?

amtyurin commented:
That’s great!

I did refresh my repo and merged old branch there.

Thanks for the advise.

Let’s move to the forum.


> On Sep 11, 2020, at 4:45 AM, Trilarion <notifications@github.com> wrote:
> 
> 
> I had done a commit on master here before you started working on fixes. I would like to keep a linear history if possible. Therefore I rebased your fixes branch against my master branch and also added another commit. This means that the histories of my master and your master are now different, while the actual content is almost the same.
> 
> I advice you to:
> 
> check that your fixes and my master is almost identical (with a directory comparison tool for example)
> reset your master to my master for example with something like
> git remote add upstream https://github.com/Trilarion/imperialism-remake.git <https://github.com/Trilarion/imperialism-remake.git>
> git reset --hard master upstream/master
> continue with a new feature branch (fixes is not needed anymore or reset fixes than to your master)
> Where do you prefer to discuss further programming? Here in the issues or on the forum at https://remake.twelvepm.de/forum/ <https://remake.twelvepm.de/forum/>?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Trilarion/imperialism-remake/pull/35#issuecomment-691045454>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAXCT4HAQUHJ2G6QF5OGZGDSFIET5ANCNFSM4Q3QZFTA>.
>
